index.html

```diff
@@ -0,0 +1,243 @@
+                        dont hesitate to contact me.</p>
+            </div>
+            
+            <div class="icons">
+                <a href="">
+                    <img src="Assets/Github.svg" alt="Github">
+                </a>
+                <a href="">
+                    <img src="Assets/Linkedin.svg" alt="Linkedin">
+                </a>
+                <a href="">
+                    <img src="Assets/Icon - Angelist.svg" alt="Angelist">
+                </a>
+                <a href="">
+                    <img src="Assets/Twitter.svg" alt="Twitter">
+                </a>
+                <a href="">
+                    <img src="Assets/Medium.svg" alt="Medium">
+                </a>
+            </div>
+        </section>
+        <section id="experience">
+            
+            <h2 class="title">My Recent Works</h2>        
+            <img class="vector" src="Assets/Vector.svg" alt="">        
+            <div>
+                <img  class="placeholder" src="Assets/Img Placeholder.svg" alt="">
+                
+                <div>
+                    <h2 class="work">Multi-Post Stories</h2>
+                </div>
+                <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="teclast">
+                    <div class="css-button">CSS</div>
+                    <div class="html-button">HTML</div>
+                    <div class="boot-button">Bootstrap</div>
+                    <div class="ruby-button">Ruby</div>
+                </div>
+                <div class="orange-button">
+                    <div class="see-project">See project</div>
+                </div>
+            </div>
+            
+            <div class="grid-container">
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>            
+
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
```

Comment:
- [ ] Kindly use **h4** instead of **h2** because it optimizes the heading of your portfolio website and enhances the user experience. The `hx` tag is specifically designed for main headings or sub-headings, providing a clear and prominent representation of the page's primary topic or purpose. By utilizing the *h4* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors. Whereas you already have used the `h2` tag on **line no, 33** and the `h3` tag as I mentioned in my above *inline comment* so that's why please maintain a good hierarchy in your development for better accessibility and SEO.

- **Kindly for more information follow [this](https://seosherpa.com/header-tags/) article.**